packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -587,7 +587,7 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
     }
 
     /**
-     * Gets or sets a boolean indicating that bone animations must be computed by the CPU (false by default)
+     * Gets or sets a boolean indicating that bone animations must be computed by the CPU (true by default)

```

Comment:
Actually the comment was right, as it is saying "CPU".

However, I think it's clearer to say "GPU" instead, and in that case `true` is the right default value.

Reply:
I agree, but you should replace "CPU" by "GPU" for the doc to be accurate.

Reply:
I applied it